algo.js

```diff
@@ -542,11 +479,12 @@ if (sendMail === "send") {
 
 // Run the Algorithm
 const mode = process.argv[2];
+
 if (mode === 'run') {
-  addMatches('Matches (Sep 19th)', jsonObj);
+  addMatches(new Date().toLocaleDateString(), prevMatches, 'MatchesTest.json', spreadSheet);
```

Comment:
This doesn't matter much but do we want to keep the format the same of 'Matches (date)'?

Reply:
the change would be to just have the date

Reply:
Ya that's fine